azure-pipelines.yml

```
@@ -122,7 +122,7 @@ stages:
       - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
         - job: OSX
           pool:
-            name: Hosted macOS
+            vmImage: 'macOS-10.15'
```

I think we can just pick `latest` (found it [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software))
```suggestion
            vmImage: 'macOS-latest'
```